frontend/src/styles/tailwind.css

```diff
@@ -11243,40 +11259,47 @@ video {
   display: contents;
 }
 
+.list-item{
```

Comment:
I am thinking now whether or not we could have not included this file and instead it be generated or something.  Too late now but something to keep in mind in case our checkout is too large to submit.